miso-web/src/main/webapp/pages/bulkEditLibraries.jsp

```diff
@@ -78,20 +78,32 @@
        <p>The following rows failed to save:</p>
        <p id="errorMessages"></p>
      </div>
-   
+
    <div id="hotContainer"></div>
- 
+
  </div>
-  
+
   <script type="text/javascript">
     jQuery(document).ready(function () {
       Library.hot.librariesJSON = ${librariesJSON};
-      Library.hot.librariesJSON = Hot.sortByProperty(Library.hot.librariesJSON, 'parentSampleAlias');
+      Hot.detailedSample = JSON.parse(document.getElementById('HOTbulkForm').dataset.detailedSample);
+
+      if (Hot.detailedSample) {
+        Library.hot.librariesJSON = Hot.sortByProperty(Library.hot.librariesJSON, 'parentSampleAlias');
+      }
+      <c:choose>
+	    <c:when test="${not empty hideCols}">
+	      Library.hot.hideCols = ${hideCols};
+	    </c:when>    
+	    <c:otherwise>
+	      Library.hot.hideCols = {};
```

Comment:
Since you're initializing `${hideCols}` as a JSONArray, you could also just make the assignment instead of wrapping it in the test; if you keep it this way, I think this needs to be an empty array instead of empty object since you're doing checks for `indexOf`.